src/index.html

```diff
@@ -11,12 +11,58 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link rel="stylesheet" href="./style.css"
```

Comment:
Keep your attributes correctly formatted. Since the link element has more than two attributes, each attribute should start on a new line with a 2-space indentation.